jujugui/static/gui/src/app/components/header-breadcrumb/header-breadcrumb.js

```diff
@@ -65,15 +65,22 @@ YUI.add('header-breadcrumb', function() {
 
     render: function() {
       var auth = this.props.authDetails;
-      var loading = auth && auth.loading;
-      var username = loading ? '...' : auth && auth.user || 'anonymous';
-      return (
-        <ul className="header-breadcrumb">
+      var userItem;
+      if (auth && (auth.user || auth.loading)) {
```

Comment:
Yeah, this is the official solution for the time being - drop the username from the breadcrumbs if we're not logged in.